ansible/templates/usb/scripts/headnode.sh.j2

```diff
@@ -396,14 +450,19 @@ function install_archives() {
 }
 
 function _zk() {
-	zkcli="${USB_COPY}/scripts/zookeepercli"
 	cmd="${@}"
 
-	if ! "${zkcli}" -servers "${CFGDB_IP}:2181" -auth_usr "esdc" -auth_pwd "${ESDC_INSTALL_PASSWORD}" -c ${cmd} &> /dev/null; then
+	if ! "${QUERY_CFGDB}" ${cmd} &> /dev/null; then
 		fatal "Error: cfgdb command \"${cmd}\" failed"
 	fi
 }
 
+test_cfgdb_connect() {
+	if [[ -z "$(_zk get /esdc)" ]]; then
```

Comment:
Querying the top-level /esdc is forbidden in our new ZK rest service, e.g. query /esdc/settings.